Added ZREMRANGEBYSCORE command

jurka opened the pull request.

jurka committed:
Added ZREMRANGEBYSCORE command

jurka committed:
typo fix in ZREMRANGEBYSCORE command defenition

jurka committed:
fixed return type in Zremrangebyscore command

jurka committed:
added ZREMRANGEBYSCORE declaration to AsyncClient interface

alphazero commented:
Hi, thank you.  I can not merge any pull requests that do not have included tests.

jurka committed:
changed precision for floats to 15 significant digits

wellle committed:
Adds automatic connection reconnecting

Adds `reconnect()` methods for `connHdl` and `asyncConnHdl` which get
called when `connHdl.ServiceRequest()` or `asyncConnHdl.QueueRequest()`
fail. Adds basic implementation of `asyncConnHdl.disconnect()`.

jurka committed:
Merge branch 'go1' of git://github.com/adeven/Go-Redis into go1

jurka committed:
decreasing DefaultReconnectDelay to half of second
